test/storage/append_events_test.exs

```diff
@@ -4,6 +4,13 @@ defmodule EventStore.Storage.AppendEventsTest do
   alias EventStore.{EventFactory, RecordedEvent, UUID}
   alias EventStore.Storage.{Appender, CreateStream}
 
+  test "append single event with a db connection error" do
```

Comment:
Not working, I need some help trying to test this thou, I am not sure what could I do to trigger the new branch

Reply:
Test fixed in b0a5e022d2d45af9dcea08ffef721a66569c4ca8.